app/queries/no_bp_measure_query.rb

```diff
@@ -12,7 +15,7 @@ def call(region, period, with_exclusions: false)
     registration_date = period.blood_pressure_control_range.begin
 
     Patient
-      .for_reports(with_exclusions: with_exclusions, exclude_ltfu_as_of: period.end_time)
+      .for_reports(exclude_ltfu_as_of: period.end_time)
```

Comment:
I'm thinking the `for_reports` should be responsible for doing the `end_time` for periods...that way callers don't need to worry about it.